client/me/sidebar/style.scss

```diff
@@ -1,19 +1,16 @@
-.is-group-me .sidebar {
-	padding-top: 16px;
-}
-
-.me-sidebar__signout {
+.sidebar__me-signout {
 	flex-shrink: 0;	// these are required for the flexed sidebar to not implode in Safari
```

Comment:
Is there a specific reason why this property isn't set in the selector below? They appear to be identical.

Reply:
Ah, strange. I was just renaming things to make the linter happy — but you're right. There's no reason to have this selector twice.